test/unit/test_utils.py

```diff
@@ -51,7 +51,7 @@ def test_get_tile_tif(self):
         if not op.isdir(tiles_dir):
             makedirs(tiles_dir)
 
-        get_tile_tif(tile, 'test/fixtures/drone.tif', dest_folder)
+        get_tile_tif(tile, 'test/fixtures/drone.tif', dest_folder, None)
```

Comment:
Is it reasonable to add an actual test for offset?

Reply:
Definitely and for posterity, this request solved a future bug 🙏